### PR TITLE
MNT: Update for conda-build 3

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've updated the recipe for conda-build 3 as instructed in #8.

List of changes done to the recipe:
Renamed build with host
Removing pinnings for numpy to use values from conda_build_config.yaml. If you need the pin see [here](https://conda-forge.org/docs/meta.html#pinning-packages) for details.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

### CODE REVIEW

**Comment by mbargull** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/T/TPOT/TPOT-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   noarch: python
```
Comment (mbargull) [inline recipe/meta.yaml]: Is the `- pywin32  # [win]` requirement necessary? If yes, please remove `noarch: python`. If not, please remove `- pywin32  # [win]`

**Review by mbargull — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (dhirschfeld) [reply recipe/meta.yaml]: I'll check that out...

Comment (dhirschfeld) [reply recipe/meta.yaml]: It seems `pywin32` is a legit dependency:
 - https://github.com/EpistasisLab/tpot/issues/333
 - https://github.com/EpistasisLab/tpot/blob/507b45db01e8f88651f4ce8e03b607a5b50146f5/tpot/base.py#L75-L94

Comment (mbargull) [reply recipe/meta.yaml]: Thanks for checking. I'll change the recipe accordingly.

**Comment by mbargull** [comment]
@conda-forge-admin, please rerender

**Review by mbargull — APPROVED** [review]

**Comment by bollwyvl** [comment]
Sorry for the radio silence, been off the grid over the long weekend. Changes look good!

**Pull request merged**